src/web/components/Links.tsx

```diff
@@ -131,78 +142,71 @@ const links = css`
     }
 `;
 
-const profileSubdomain = 'https://profile.theguardian.com';
-const jobsUrl = 'https://jobs.theguardian.com/?INTCMP=jobs_uk_web_newheader';
-const datingUrl =
-    'https://soulmates.theguardian.com/?INTCMP=soulmates_uk_web_newheader';
-const signInUrl = `${profileSubdomain}/signin?INTCMP=DOTCOM_NEWHEADER_SIGNIN&ABCMP=ab-sign-in`;
-
 const identityLinks: DropdownLinkType[] = [
     {
-        url: `${profileSubdomain}/user/id/123`, // TODO use actual user ID once we have a user model
+        url: `https://profile.theguardian.com/user/id/123`, // TODO use actual user ID once we have a user model
```

Comment:
Should this still be a dummy or undefined?

Reply:
I think what we are going to need to do is make an api call after page load to get the signed in user's profile, and pass that down. In general, there's a lot of identity logic being used by FE and discussion that we need to review.